Sources/StreamChatUI/Common Views/ChatChannelCreateNewButton_Tests.swift

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelCreateNewButton_Tests: XCTestCase {
+    func test_defaultAppearance() {
+        let view = ChatChannelCreateNewButton().withoutAutoresizingMaskConstraints
+        AssertSnapshot(view)
+    }
+    
+    func test_isHighlighted() {
+        let view = ChatChannelCreateNewButton().withoutAutoresizingMaskConstraints
+        view.isHighlighted = true
+        AssertSnapshot(view)
+    }
+    
+    func test_isDisabled() {
+        let view = ChatChannelCreateNewButton().withoutAutoresizingMaskConstraints
+        view.isEnabled = false
+        AssertSnapshot(view)
+    }
+
+    func test_customizationUsingAppearanceHook() {
+        class TestView: ChatChannelCreateNewButton {}
+        TestView.defaultAppearance {
+            // Modify appearance
+            $0.layer.borderWidth = 1
+            $0.layer.borderColor = UIColor.green.cgColor
+            $0.backgroundColor = .black
+            $0.tintColor = .lightGray
+        }
```

Comment:
it'd be nice to add a couple of constraints here to modify the size of the button, too

Reply:
✅ Done